openesdh-doc-templates/src/main/java/dk/openesdh/doctemplates/services/documenttemplate/DocumentTemplateServiceImpl.java

```diff
@@ -1,4 +1,4 @@
-package dk.openesdh.repo.services.documents;
```

Comment:
I guess we need some sort of security proxies for document templates services to check whether a tenant has permissions to call services. See the TenantOpeneModuleService.checkCurrentTenantModuleEnabled()